{{ .ProjectSnake }}/requirements/all.in

```diff
@@ -0,0 +1,4 @@
+# Collect all dependencies, to pass to pip.parse and create the @pip repository.
```

Comment:
On further consideration, I agree with https://github.com/bazelbuild/rules_python/issues/1971#issuecomment-2184134902. Probably worth adding a comment that this is a workaround for that limitation, and in the meantime can result in otherwise-avoidable dependency conflicts.